topology/Arizona State University/ASU/DELL_M420_ASU.yaml

```diff
@@ -0,0 +1,31 @@
+Disable: false
+Production: true
+SupportCenter: Self Supported
+GroupID: 486
+GroupDescription: Arizona State University Research Computing.
+Resources:
+  DELL_M420_ASU:
```

Comment:
@zvada requested that we change this to `OSG_DELL_M420_ASU`

Reply:
OK sure